# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1437068991 14400
#      Thu Jul 16 13:49:51 2015 -0400
# Node ID bf92a14c0e7638c1f5001ab39a14196a32bc7da4
# Parent  fb97c11f8e676d594aba630f1e57bdf9b813fc11
don't include GEN_FCN_FILES in FCN_FILES list

* scripts/module.mk (GEN_FCN_FILES_IN): Rename from FCN_FILES_IN.
* scripts/plot/util/module.mk (FCN_FILES, EXTRA_DIST): Remove
$(GEN_FCN_FILES) from the lists.

diff --git a/scripts/module.mk b/scripts/module.mk
--- a/scripts/module.mk
+++ b/scripts/module.mk
@@ -60,17 +60,17 @@ include scripts/statistics/distributions
 include scripts/statistics/models/module.mk
 include scripts/statistics/tests/module.mk
 include scripts/strings/module.mk
 include scripts/testfun/module.mk
 include scripts/time/module.mk
 
 image_DATA += $(SCRIPTS_IMAGES)
 
-FCN_FILES_IN = $(GEN_FCN_FILES:.m=.in)
+GEN_FCN_FILES_IN = $(GEN_FCN_FILES:.m=.in)
 
 ALL_LOCAL_TARGETS += \
   $(GEN_FCN_FILES) \
   $(PKG_ADD_FILES) \
   $(JAR_FILES)
 
 if AMCOND_BUILD_DOCS
 ALL_LOCAL_TARGETS += scripts/.DOCSTRINGS
@@ -198,18 +198,17 @@ scripts-dist-hook:
 else
 scripts-dist-hook:
 	@echo "Packaging distribution requires Java." ; exit 1;
 endif
 
 EXTRA_DIST += \
   $(SCRIPTS_IMAGES) \
   $(FCN_FILES) \
-  $(FCN_FILES_IN) \
-  $(GEN_FCN_FILES) \
+  $(GEN_FCN_FILES_IN) \
   scripts/DOCSTRINGS \
   scripts/mkdoc.pl \
   scripts/mk-pkg-add
 
 DISTCLEANFILES += \
   scripts/.DOCSTRINGS \
   scripts/DOCSTRINGS \
   $(PKG_ADD_FILES) \
diff --git a/scripts/plot/util/module.mk b/scripts/plot/util/module.mk
--- a/scripts/plot/util/module.mk
+++ b/scripts/plot/util/module.mk
@@ -88,15 +88,14 @@ scripts_plot_util_DATA = \
   $(scripts_plot_util_GEN_FCN_FILES)
 
 scripts_plot_util_privatedir = $(fcnfiledir)/plot/util/private
 
 scripts_plot_util_private_DATA = $(scripts_plot_util_PRIVATE_FCN_FILES)
 
 FCN_FILES += \
   $(scripts_plot_util_FCN_FILES) \
-  $(scripts_plot_util_GEN_FCN_FILES) \
   $(scripts_plot_util_PRIVATE_FCN_FILES)
 
 PKG_ADD_FILES += scripts/plot/util/PKG_ADD
 
 DIRSTAMP_FILES += scripts/plot/util/$(octave_dirstamp)
 
